yash/tests/scripted_test/kill1-p.sh

```diff
@@ -0,0 +1,55 @@
+# kill1-p.sh: test of the kill built-in for any POSIX-compliant shell, part 1
+
+posix="true"
+
+test_E -e 0 'printing all signal names'
```

Comment:
Calling `test_printing_signal_name_from_number` inside a test case would not make sense.
We don't test the `SIGBUS` signal because it does not have a portable signal number defined in POSIX.